lib/rspecq/worker.rb

```diff
@@ -194,6 +234,9 @@ def try_publish_queue!(queue)
         # HEURISTIC: put jobs without previous timings (e.g. a newly added
         # spec file) in the middle of the queue
         h[j] = timings[j] || default_timing
+        if j.include? "["
+          h[j] = 999_999 # force to put example jobs in front
```

Comment:
ruby doesn't have something like Number.MAX_SAFE_INTEGER 🤔

Reply:
I don't think ruby has any constant variable like that, unless we use something like [this](https://gist.github.com/pithyless/9738125), but overkill 😂